values/strings.xml

```diff
@@ -1385,9 +1373,6 @@ If you provide TheoTown access to external storage, it will save cities in the d
     <string name="draft_destroyedtileradioactive00_title">Scrap</string>
     <string name="draft_destroyedtileradioactive00_text">A radioactive destroyed structure. You better remove it.</string>
     <string name="settings_disaster_ufo">UFO</string>
-    <string name="dialog_externalstorage_text">TheoTown needs access to external storage to save your progress.</string>
-    <string name="dialog_externalstorage_cancel">Cancel</string>
-    <string name="dialog_externalstorage_retry">Retry</string>
```

Comment:
The game might still make calls to legacy APIs and request the permission via such dialog